### PR TITLE
fix: removing empty keys from value skeleton

### PR DESCRIPTION
**Pull request by mojtabaimani**
Empty keys in value skeleton makes validation problem after merging with otomi chart values if user doesn't provide values for those empty keys.

### CODE REVIEW

**Review by Dunky13 — APPROVED** [review]
Lgtm

File: values/otomi-api/otomi-api.gotmpl [hunk 1]
```diff
@@ -43,7 +43,7 @@ env:
   GIT_REPO_URL: {{ $o | get "git.repoUrl" $giteaValuesUrl }}
   GIT_BRANCH: {{ $o | get "git.branch" "main" }}
   CLUSTER_ID: {{ printf "%s/%s" $c.provider $c.name }}
-  CLUSTER_NAME: {{ $c.apiName }}
+  CLUSTER_NAME: {{ $c | get "apiName" "api" }}
```
Comment (j-zimnowoda) [inline values/otomi-api/otomi-api.gotmpl:46]: The otomi-api does not use CLUSTER_NAME anymore

File: values/otomi-api/otomi-api.gotmpl [hunk 2]
```diff
@@ -43,7 +43,7 @@ env:
   GIT_REPO_URL: {{ $o | get "git.repoUrl" $giteaValuesUrl }}
   GIT_BRANCH: {{ $o | get "git.branch" "main" }}
   CLUSTER_ID: {{ printf "%s/%s" $c.provider $c.name }}
```
Comment (j-zimnowoda) [inline values/otomi-api/otomi-api.gotmpl:45]: The otomi-api does not use CLUSTER_ID anymore

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]

File: values/otomi-api/otomi-api.gotmpl [hunk 3]
```diff
@@ -43,7 +43,7 @@ env:
...
-  CLUSTER_NAME: {{ $c.apiName }}
+  CLUSTER_NAME: {{ $c | get "apiName" "api" }}
   CLUSTER_APISERVER: {{ $c.apiServer }}
```
Comment (j-zimnowoda) [inline values/otomi-api/otomi-api.gotmpl:47]: ditto

**Review by j-zimnowoda — APPROVED** [review]

**Pull request merged**